src/bun.js/bindings/bindings.cpp

```diff
@@ -229,6 +229,45 @@ AsymmetricMatcherResult matchAsymmetricMatcher(JSGlobalObject* globalObject, JSC
             }
         }
 
+        return AsymmetricMatcherResult::FAIL;
+    } else if (auto* expectArrayContaining = jsDynamicCast<JSExpectArrayContaining*>(matcherPropCell)) {
+        JSValue expectedArrayValue = expectArrayContaining->m_arrayValue.get();
+
+        if (otherProp.isCell() && otherProp.asCell()->type() == ArrayType) {
```

Comment:
There are a few other types of arrays this could be, like `DerivedArray`

Instead, for checking if a JSValue is an array, we should use `JSC::isJSArray`

Reply:
ah turns out that method actually does the same thing

we should still use it though incase JSC makes a change in the future